bitnami/kafka/templates/kafka-metrics-role.yaml

```
@@ -0,0 +1,23 @@
+{{- if .Values.rbac.create -}}

```

This condition is incomplete
```suggestion
{{- if and .Values.rbac.create .Values.metrics.kafka.enabled Values.metrics.kafka.serviceAccount.create  }}
```

---

Good catch I totally missed that